src/cmd-buildextend-installer

```diff
@@ -143,6 +151,19 @@ def cp_reflink(src, dest):
     subprocess.check_call(['cp', '--reflink=auto', src, dest])
 
 
+# Make stream hash for `coreos-installer stream-hash`
+def make_stream_hash(src, dest):
+    bufsize = 2 * 1024 * 1024
+    with open(src, 'rb') as inf:
+        with open(dest, 'w') as outf:
+            outf.write('stream-hash sha256 {}\n'.format(bufsize))
```

Comment:
`stream-hash sha256 {}` is just information we're putting at the top of the file right? Should we include the assumed bufsize here as well in case we change it in the future?

Reply:
Yup, that's what `'{}'.format(bufsize)` is doing. :slightly_smiling_face:

Reply:
sigh.. i totally missed that. haha, i've been staring at code too much.